plugin.py

```diff
@@ -142,11 +148,11 @@ def UpdateDevice(self, lune, luneage):
                 datafr = "MoonPhases8WC"
                 phase = 8
 
-
-            Domoticz.Debug("Icon " + str(datafr))
+            if self.southern_hemi:
+                datafr = '%s_%s' % (self.prefix,datafr)
+            Domoticz.Debug("Setting Icon " + str(datafr))
             try:
-               #Devices[1].Update(0, str(lune))
-               Devices[1].Update(nValue=0, sValue=str(luneage), Image=Images[datafr].ID)
+               Devices[1].Update(nValue=0, Name=str(lune), sValue=str(luneage), Image=Images[datafr].ID)
```

Comment:
Am not sure if its wise to change device name all the time. Am afraid this way you cannot use this device to scripts/blockly etc.

Reply:
Hard to find any documentation on the various desktop types and the C code gives me a headache. Need a type that allows the phase and days to be displayed cleanly. Suggestions welcomed. Otherwise, it's working the way I like it and I have no plans for scripting so I'll await further suggestions....